clients/drcachesim/reader/record_file_reader.h

```diff
@@ -235,6 +235,11 @@ class record_reader_t : public reader_base_t {
     {
         return in_kernel_trace_;
     }
+    uint64_t
+    get_next_trace_pc() const override
```

Comment:
See other comment: this can be in the base class.

Reply:
Done.